torchvision/datasets/cityscapes.py

```diff
@@ -191,3 +203,9 @@ def _get_target_suffix(self, mode, target_type):
             return '{}_color.png'.format(mode)
         else:
             return '{}_polygons.json'.format(mode)
+
+
+def extract_cityscapes_zip(zip_location, root):
```

Comment:
couldn't you use https://github.com/pytorch/vision/blob/867e5b4e7835dbebcf0c501850a7b08e466496f1/torchvision/datasets/utils.py#L215-L236 instead ?